deploy/tasks/sentry_notify.php

```diff
@@ -0,0 +1,186 @@
+<?php
```

Comment:
Why not keep this in the already existing `deploy/tasks/sentry.php` file?

Reply:
Actually, even better, in `overrides.php` file which was meant for overrides of existing tasks from recipes :)

Reply:
Updated in https://github.com/netgen/media-site/pull/90/commits/b16cc52a0abc7bd681bbb73fbe3f58f630e77724